services/QuillLMS/lib/tasks/vitally.rake

```diff
@@ -42,7 +42,7 @@ namespace :vitally do
             messageId: SecureRandom.uuid
           }
         end
-      end.flatten
+      end.flatten.reject(&:nil?)
```

Comment:
If you want to play code golf: `end.flatten.compact`
But this is a rake task so not a big deal.